backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ProjectDatabaseHandler.java

```diff
@@ -1632,4 +1635,276 @@ public ProjectData searchByTag(String tag, User user) {
     public ProjectData searchByType(String type, User user) {
         return repository.searchByType(type, user);
     }
+
+
+
+    /*
+     * This function will get list ReleaseLink from release network of project with trace from root to clicked release on screen
+     * Trace is list index of each node.
+     */
+    public List<ReleaseLink> getReleaseLinksOfProjectNetWorkByTrace(List<String> trace, String projectId, User user){
+        Project project = repository.get(projectId);
+        String releaseNetwork = project.getReleaseRelationNetwork();
+        ObjectMapper mapper = new ObjectMapper();
+        List<ReleaseLinkJSON> listReleaseLinkJson;
+        List<ReleaseLink> linkedReleases = new ArrayList<>();
+        try {
+            listReleaseLinkJson = mapper.readValue(releaseNetwork, new TypeReference<List<ReleaseLinkJSON>>() {
```

Comment:
Modify <List<ReleaseLinkJSON>> to <>

IDE note: Explicit type arguments <List<ReleaseLinkJSON>> can be replaced with <>